### PR TITLE
Convert all access transformers to Mixin accessors

### PR DESCRIPTION
**Pull request by ChachyDev**

### CODE REVIEW

File: src/main/resources/patcher.mixins.json [hunk 1]
```diff
@@ -120,5 +126,8 @@
     "performance.network.packet.C17PacketCustomPayloadMixin_MemoryLeak",
     "performance.network.packet.S3FPacketCustomPayloadMixin_MemoryLeak"
   ],
-  "plugin": "club.sk1er.patcher.mixins.plugin.PatcherMixinPlugin"
+  "plugin": "club.sk1er.patcher.mixins.plugin.PatcherMixinPlugin",
+  "mixins": [
+    "accessors.BlockAccessor"
```
Comment (asbyth) [inline src/main/resources/patcher.mixins.json:131]: Shove this into the client array

File: src/main/resources/patcher_at.cfg [hunk 2]
```diff
@@ -1,124 +1,3 @@
...
-public net.minecraft.client.renderer.tileentity.TileEntityBannerRenderer field_178466_c # DESIGNS
-public net.minecraft.client.renderer.tileentity.TileEntityBannerRenderer field_178464_d # BANNERTEXTURES
-public net.minecraft.client.renderer.tileentity.TileEntityBannerRenderer$TimedBannerTexture
-public net.minecraft.client.renderer.tileentity.TileEntityBannerRenderer$TimedBannerTexture <init>()V # constructor
-
-# GuiSelectWorld
-public net.minecraft.client.gui.GuiSelectWorld field_146632_a # parentScreen
-
-# BlockRedstoneTorch
+public net.minecraft.client.renderer.GlStateManager$Color
```
Comment (asbyth) [inline src/main/resources/patcher_at.cfg:2]: Move this above TextureState so the diff is more precise

File: src/main/resources/patcher.mixins.json (hunk 1 above)
Comment (LlamaLad7) [reply src/main/resources/patcher.mixins.json:131]: MCDev moment

**Review by LlamaLad7 — APPROVED** [review]
Looks fine at a glance.

File: src/main/java/club/sk1er/patcher/hooks/MinecraftHook.java [hunk 3]
```diff
@@ -23,9 +25,10 @@
  * Used in {@link MinecraftTransformer#transform(ClassNode, String)}
  */
 @SuppressWarnings("unused")
-public class MinecraftHook {
+public class    MinecraftHook {
```
Comment (asbyth) [inline src/main/java/club/sk1er/patcher/hooks/MinecraftHook.java:28]: Remove this random spacing

File: src/main/java/club/sk1er/patcher/hooks/ResourcePackRepositoryHook.java [hunk 4]
```diff
@@ -11,15 +12,17 @@
 
 @SuppressWarnings("unused")
 public class ResourcePackRepositoryHook {
-    public static void updateRepositoryEntriesAll(ResourcePackRepository repository) {
+    public static void updateRepositoryEntriesAll(ResourcePackRepository r) {
```
Comment (asbyth) [inline src/main/java/club/sk1er/patcher/hooks/ResourcePackRepositoryHook.java:15]: Rename this back to "repository" & make the variable accessor or something similar

File: src/main/resources/patcher_at.cfg [hunk 5]
```diff
@@ -1,124 +1,10 @@
...
-public net.minecraft.client.renderer.GlStateManager$Color field_179194_c # blue
-public net.minecraft.client.renderer.GlStateManager$Color field_179192_d # alpha
-
-# GlStateManager$TextureState
 public net.minecraft.client.renderer.GlStateManager$TextureState
-public net.minecraft.client.renderer.GlStateManager$TextureState field_179059_b # textureName
-
-# Block
-public net.minecraft.block.Block field_149756_F # maxY
+public net.minecraft.block.BlockRedstoneTorch$Toggle
```
Comment (asbyth) [inline src/main/resources/patcher_at.cfg:3]: Move this to the bottom & add it's comment back so the diff is more precise

File: src/main/resources/patcher_at.cfg [hunk 6]
```diff
@@ -1,124 +1,10 @@
...
-public net.minecraft.client.renderer.entity.RenderManager field_78726_c # renderPosY
-public net.minecraft.client.renderer.entity.RenderManager field_78723_d # renderPosZ
-
-# Item
-public net.minecraft.item.Item field_111210_e # itemModifierUUID
-
-# ResourcePackRepository
-public net.minecraft.client.resources.ResourcePackRepository field_110619_e # repositoryEntriesAll
-public net.minecraft.client.resources.ResourcePackRepository func_110614_g()Ljava/util/List; # getResourcePackFiles
+# net.minecraft.client.resources.ResourcePackRepository$Entry
```
Comment (asbyth) [inline src/main/resources/patcher_at.cfg:5]: Rename this to `ResourcePackRepository`

File: src/main/resources/patcher_at.cfg [hunk 7]
```diff
@@ -1,124 +1,10 @@
...
 public net.minecraft.client.resources.ResourcePackRepository$Entry <init>(Lnet/minecraft/client/resources/ResourcePackRepository;Ljava/io/File;)V # constructor
 
-# GuiChat
-public net.minecraft.client.gui.GuiNewChat field_146252_h # chatLines
-public net.minecraft.client.gui.GuiNewChat field_146253_i # drawnChatLines
-
-# TileEntityBannerRenderer
-public net.minecraft.client.renderer.tileentity.TileEntityBannerRenderer field_178466_c # DESIGNS
-public net.minecraft.client.renderer.tileentity.TileEntityBannerRenderer field_178464_d # BANNERTEXTURES
+# net.minecraft.client.renderer.tileentity.TileEntityBannerRenderer$TimedBannerTexture
```
Comment (asbyth) [inline src/main/resources/patcher_at.cfg:8]: Rename this to `TileEntityBannerRenderer`

**Review by asbyth — CHANGES_REQUESTED** [review]

**Pull request merged**